source/_components/trafikverket_train.markdown

```diff
@@ -0,0 +1,118 @@
+---
+layout: page
+title: "Trafikverket Train"
+description: "Instructions how to integrate Trafikverket Train within Home Assistant."
+date: 2019-04-27 16:32
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: trafikverket.png
+ha_category:
+  - Transport
+  - Sensor
+ha_release: 0.93
+ha_iot_class: Cloud Polling
+---
+
+Retreive train departure information from [Trafikverket](https://www.trafikverket.se/).
+
+Use cases:
+
+- Retreive the next departure and information connected to it for a specific train line.
+- Retreive information for a specific departure time on a specific train line.
+- Set up alert or perform actions if your train is delayed or cancelled.
+
+Data that is retreived: 
+
+- Next departure for specific train line.
+- Canceled status.
+- Number of minutes delayed.
+- Planned time if no delays occur.
+- Estimated time of arrival if delays occur.
+- Actual time - when it did arrive.
+- Other information / additional texts.
+- Deviations.
+
+## {% linkable_title Configuration %}
+
+To enable this sensor, use the following format in your `configuration.yaml`.
+
+```yaml
+sensor:
```

Comment:
Only required items in the first example, `weekday` should not be included in this example. You can add it to your example further below.

Reply:
Done